src/openrct2/ride/ride.h

```diff
@@ -80,16 +80,26 @@ typedef struct vehicle_colour_preset_list {
 } vehicle_colour_preset_list;
 assert_struct_size(vehicle_colour_preset_list, (1 + 256 * 3));
 
+typedef struct rct_ride_name {
+    rct_string_id name;
+    rct_string_id description;
+} rct_ride_name;
+
```

Comment:
Unless its used in an `s4` or `s6` struct, or inside an `RCT2` interop define block, it does not need user defined packing.